pyrobosim/examples/test_world.py

```diff
@@ -0,0 +1,139 @@
+"""
+Unit tests for core world modeling
+"""
+import os
+import pytest
+import numpy as np
+
+from pyrobosim.core.world import World
+from pyrobosim.core.world import Room
+from pyrobosim.utils.pose import Pose
+from pyrobosim.core.world import Object
+from pyrobosim.core.world import Hallway
```

Comment:
Do these imports actually work? It should be something like

```suggestion
from pyrobosim.core.hallway import Hallway
from pyrobosim.core.objects import Object
from pyrobosim.core.room import Room
from pyrobosim.core.world import World
from pyrobosim.utils.pose import Pose
```

Reply:
The tests are passing , so  I guess it is actually working when importing like that,  definitely weird !